### PR TITLE
feat: add query string representation

### PR DESCRIPTION
**Pull request by sashamelentyev**

### CODE REVIEW

**Comment by sashamelentyev** [comment]
@vmihailenco i did it

File: query_select.go [hunk 1]
```diff
@@ -920,6 +920,17 @@ func (q *SelectQuery) whereExists(ctx context.Context) (bool, error) {
 	return n == 1, nil
 }
 
+func (q *SelectQuery) String() (string, error) {
```
Comment (vmihailenco) [inline query_select.go:923]: Let's just return `string` so we satisfy `Stringer` interface. We can panic on errors.

File: query_select.go [hunk 2]
```diff
@@ -920,6 +920,17 @@ func (q *SelectQuery) whereExists(ctx context.Context) (bool, error) {
...
+	var buf []byte
```
Comment (vmihailenco) [inline query_select.go:924]: Nitpick: no need to define `buf`.

**Comment by sashamelentyev** [comment]
@vmihailenco done

**Comment by vmihailenco** [comment]
Thank you :+1:

**Pull request merged**